abb_experimental/package.xml

```diff
@@ -8,9 +9,13 @@
   <author email="sedwards@swri.org">Shaun Edwards</author>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <run_depend>abb_irb4400_support</run_depend>
+
+  <exec_depend>abb_resources</exec_depend>
+  <exec_depend>abb_driver</exec_depend>
```

Comment:
A bit late perhaps but it's not customary to make regular metapackages depend on packages in other repositories. Besides, the `abb_driver` and `abb_resources` packages are already dependencies of the packages in this repository, so they should be resolved and installed by tools like `rosdep` automatically.